src/input/input.directive.ts

```diff
@@ -17,3 +17,13 @@ import { Directive, HostBinding } from "@angular/core";
 export class TextInput {
 	@HostBinding("class") inputClass = "bx--text-input";
 }
+
+
+@Directive({
```

Comment:
This directive should be in a separate `text-area.directive.ts`

I'd say separate module, but since both it and `input.directive` are related to text entry we're probably fine to leave it here.

Reply:
Sure- Wasn't sure when to separate it out in source, I was thinking of this as related to text input because of the way carbon organizes their docs